packages/engine-rn-next/src/tasks/task.rnv.export.ts

```diff
@@ -1,22 +1,11 @@
-import {
-    RnvTaskFn,
-    logErrorPlatform,
-    logTask,
-    WEB,
-    CHROMECAST,
-    TASK_BUILD,
-    TASK_EXPORT,
-    PARAMS,
-    executeOrSkipTask,
-    shouldSkipTask,
-} from '@rnv/core';
+import { RnvTaskFn, logErrorPlatform, logTask, WEB, CHROMECAST, TASK_EXPORT, PARAMS, shouldSkipTask } from '@rnv/core';
 import { exportWebNext } from '../sdk';
 
 export const taskRnvExport: RnvTaskFn = async (c, parentTask, originTask) => {
     logTask('taskRnvExport', `parent:${parentTask}`);
     const { platform } = c;
 
-    await executeOrSkipTask(c, TASK_BUILD, TASK_EXPORT, originTask);
+    // await executeOrSkipTask(c, TASK_BUILD, TASK_EXPORT, originTask);
```

Comment:
@mihaiblaga89 commenting out this breaks chained execution. if you want to skip TASK_BUILD you still need to call task which propagates under TASK_BUILD.

in this case:

```
    await executeOrSkipTask(c, TASK_PACKAGE, TASK_BUILD, originTask);
```